README.md

```diff
@@ -76,3 +76,11 @@ Run the following commands:
 cd /usr/share/cephmetrics-ansible
 ansible-playbook -v -i ~/cephmetrics/inventory -e '@~/cephmetrics/vars.yml' playbook.yml
 ```
+## Known Issues
+1. The dashboard queries rely on a template variable called domain which provides the dns
+domain name for the ceph hosts. If your environment is not using dns, the collectd data
+sent to graphite will not include a dns suffix in the metric name. Without the dns suffix
+the dashboard queries fail, and your dashboard panels show N/A.  
+ **Workaround**: On each of your ceph nodes, update collectd.conf to use a fully qualified 
+  domain name on the HostName parameter. This will ensure the domain name component
```

Comment:
This will be overridden if the deploy is run a second time, since it writes `collectd.conf`.

We could set `HostName` in `collectd.conf` to the FQDN during deployment. If there is no DNS for the cluster nodes, this remains a sticky issue though. Is having DNS entries somehow an unacceptable requirement?